src/main/java/uk/gov/hmcts/reform/civil/service/EventEmitterService.java

```diff
@@ -23,20 +24,36 @@ public void emitBusinessProcessCamundaEvent(CaseData caseData, boolean dispatchP
         var businessProcess = caseData.getBusinessProcess();
         var camundaEvent = businessProcess.getCamundaEvent();
         log.info(format("Emitting %s camunda event for case: %d", camundaEvent, caseId));
+
+        boolean nullTenantAttempt = false;
         try {
             runtimeService.createMessageCorrelation(camundaEvent)
+                .tenantId(TENANT_ID)
                 .setVariable("caseId", caseId)
                 .correlateStartMessage();
-
             if (dispatchProcess) {
                 applicationEventPublisher.publishEvent(new DispatchBusinessProcessEvent(caseId, businessProcess));
             }
-
-            log.info("Camunda event emitted successfully");
+            log.info("Camunda event emitted successfully with tenant");
         } catch (Exception ex) {
-            log.error(format("Emitting %s camunda event failed for case: %d, message: %s",
-                             camundaEvent, caseId, ex.getMessage()
-            ));
+            nullTenantAttempt = true;
```

Comment:
Do we know any specific exception which is thrown in case tenant id attempts fails because of reason of not able to find the process and can we then catch that specific exception here?